components/ArticleList.jsx

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useState } from "react";
+import ArticleCard from "./ArticleCard";
+import axios from "axios";
+
+const ArticleList = () => {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    axios.get("https://newssite-zy4v.onrender.com/api/articles").then((res) => {
```

Comment:
It would be nice if you extracted your axios requests to a function in a utils file and then import/export each function when needed. This would make your code more dry in the long run. 

If you have your axios requests in one file you could also use use axios.create to set up a base URL